substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/ProgressReporter.java

```diff
@@ -77,6 +77,7 @@
     private static final int CHARACTERS_PER_LINE;
     private static final int PROGRESS_BAR_START = 30;
     private static final boolean IS_CI = System.console() == null || System.getenv("CI") != null;
+    boolean IS_DUMB_TERM = "dumb".equals(System.getenv("TERM"));
```

Comment:
I'm going to append `private static final` to this for consistency.

Reply:
Yes, apologies for not doing so in the first place.